featurebyte/models/observation_table_tile_cache.py

```diff
@@ -1,36 +0,0 @@
-"""
```

Comment:
No longer needed as permanent tile tables are not created for historical features.